runner/test/cases/server/phase-banner.test.js

```diff
@@ -0,0 +1,80 @@
+import Lab from "@hapi/lab";
+import { expect } from "@hapi/code";
+import cheerio from "cheerio";
+import createServer from "src/server";
+
+const { test, suite, afterEach } = (exports.lab = Lab.script());
+
+suite(`Phase banner`, () => {
+  let server;
+
+  afterEach(async () => {
+    await server.stop();
+  });
+
+  test("shows the beta tag by default", async () => {
+    // For backwards-compatibility, as the main layout template currently always shows 'beta'.
+    // TODO: default to no phase banner? TBD
```

Comment:
I had a heated discussion about this a little while ago.. but imo I think it should default to alpha or beta, until there's clear guidance on forms (vs services). Most digital services in government aren't "live" (i.e. no banner required) - so it's easier/less misleading to the user that this isn't a finished and polished product